client/src/components/CommentApp/main.tsx

```diff
@@ -204,6 +205,20 @@ export class CommentApp {
     this.store.dispatch(setFocusedComment(commentId, { updatePinnedComment: true }));
     return commentId;
   }
+  invalidateContentPath(contentpath: string) {
```

Comment:
This feels like it could be an action, rather than dispatching lots of sequential actions (which we can't always avoid, but would be nice to where possible). 

If we do leave it as a function that creates lots of actions, we already have a selector for contentpath, so should probably use that rather than relying on the exact shape of the state here

Reply:
This makes sense, I'll change it, thanks!

Reply:
Done!